lib/node_modules/@stdlib/array/fixed-endian-factory/lib/main.js

```diff
@@ -569,6 +569,37 @@ function factory( dtype ) { // eslint-disable-line max-lines-per-function, stdli
 		return true;
 	});
 
+	/**
```

Comment:
Please move this function so that prototype methods are defined in alphabetical order.